docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md

```diff
@@ -0,0 +1,131 @@
+# Getting Started with Open Enclave on Windows for systems with support for SGX1 with Flexible Launch Control(FLC)
+
+## Platform requirements
+
+Intel® X86-64bit architecture with SGX1 and Flexible Launch Control (FLC) support. (e.g. Intel Coffee Lake CPU)
```

Comment:
Why do we randomly use a (R) symbol here for Intel but not elsewhere? Also "e.g." isn't great for ESL readers, might as well replace with "For example,"